pyi.py

```diff
@@ -2084,3 +2131,4 @@ def parse_options(
     "Y054 Numeric literals with a string representation "
     ">10 characters long are not permitted"
 )
+Y055 = "Y055 Multiple type builtins in a union. {suggestion}"
```

Comment:
The suggestions are great, but the description of "what's wrong" could be clearer. Maybe:

```suggestion
Y055 = 'Y055 Multiple "type[C]" members in a union. {suggestion}'
```

Reply:
This would make me wonder what `C` is. Maybe:

```suggestion
Y055 = 'Y055 Multiple "type[Foo]" members in a union. {suggestion}'
```

Reply:
"Foo" works for me!